websites/I/IFTTT/presence.ts

```diff
@@ -1,277 +1,361 @@
 const presence = new Presence({
     clientId: "665519810054062100"
   }),
-  strings = presence.getStrings({
-    search: "presence.activity.searching",
-    browsing: "presence.activity.browsing",
-    reading: "presence.activity.reading"
-  });
+  strings = getStrings();
```

Comment:
Handle language changing & define strings in the event instead of awaiting it each time